### PR TITLE
Updating_man

### PR DESCRIPTION
**Pull request by byteninjaa0**
# SQUASH ME

**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [ ] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [ ] I've documented or updated the documentation of every function and struct this PR changes. If not so I've explained why.
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**
updated the man pages of rizin.1 and rz-diff.1

...

**Test plan**

...

**Closing issues**

...

### CODE REVIEW

File: binrz/man/rz-diff.1 [hunk 1]
```diff
@@ -1,82 +1,82 @@
...
-Graph diff output of given symbol, or between two functions, at given offsets: one for each binary.
-.It Fl G [cmd]
-Run an rz command on every RzCore instance created
+Disable the use of colors in the output.
+.It Fl d Ar [algo]
+Compute the edit distance between files using the chosen algorithm:
+              myers | Eugene W. Myers' O(ND) algorithm (no substitution)
+              leven | Levenshtein O(N^2) algorithm (with substitution)
+.It
+				     ssdeep | Context triggered piecewise hashing comparison
```
Comment (wargio) [inline binrz/man/rz-diff.1]: these spaces looks weird

File: binrz/man/rz-diff.1 [hunk 2]
```diff
@@ -1,82 +1,81 @@
-.Dd Feb 10, 2018
+.Dd  10, 2018
```
Comment (XVilka) [inline binrz/man/rz-diff.1]: Broken date

File: binrz/man/rizin.1 [hunk 3]
```diff
@@ -1,132 +1,122 @@
-.Dd May 17, 2020
+.Dd 24 Aug, 2023
```
Comment (XVilka) [inline binrz/man/rizin.1]: Please use the same format

File: binrz/man/rizin.1 [hunk 4]
```diff
@@ -1,132 +1,122 @@
...
-.It Fl k Ar kernel
-Select kernel (asm.os) for syscall resolution
-.It Fl l Ar plugfile
-Load given plugin file
+Set block size to be equal to file size.
+.It -F [binplugin]
+Forces the use of a specific rbin plugin.
+.It Fl h, hh
+Shows help messages, with -hh showing detailed help.
+.It Fl H ([var])
```
Comment (XVilka) [inline binrz/man/rizin.1]: Probably worth noting that just `-H` without any argument prints useful information about the installation - location of various paths - config file, plugins, etc:
```
rizin -H
RZ_VERSION=0.7.0
RZ_PREFIX=/Users/user/.local
RZ_EXTRA_PREFIX=
RZ_MAGICPATH=/Users/user/.local/share/rizin/magic
RZ_INCDIR=/Users/user/.local/include/librz
RZ_LIBDIR=/Users/user/.local/lib
RZ_SIGDB=/Users/user/.local/share/rizin/sigdb
RZ_EXTRA_SIGDB=
RZ_LIBEXT=dylib
RZ_RCONFIGHOME=/Users/user/.config/rizin
RZ_RDATAHOME=/Users/user/.local/share/rizin
RZ_RCACHEHOME=/Users/user/.cache/rizin
RZ_LIBR_PLUGINS=/Users/user/.local/lib/rizin/plugins
RZ_EXTRA_PLUGINS=
RZ_USER_PLUGINS=/Users/user/.local/lib/rizin/plugins
RZ_IS_PORTABLE=0
```

File: binrz/man/rizin.1 [hunk 5]
```diff
@@ -1,132 +1,122 @@
...
-.It Fl nn
-Only load the rbin structures (elf, mach0, ...)
+Disable demangling.
+.It Fl n, nn
+Controls loading of RzBin info, with -nn only loading binary structures.
 .It Fl N
 Do not load user settings/projects from ~/.rizinrc, ~/.config/rizin/rizinrc and the scripts inside .../rizinrc.d/ directory.
 .It Fl NN
-Same as -N but also disables the automatic loading of plugins on startup time
+Disables loading any scripts or plugins.
```
Comment (XVilka) [inline binrz/man/rizin.1]: Please be consistent. Sometimes it `disable`, sometimes it `disables`. I suggest follow the same style everywhere.

**Review by XVilka — CHANGES_REQUESTED** [review]

**Review by wargio — APPROVED** [review]
LGTM besides an extra space.

File: binrz/man/rz-diff.1 [hunk 6]
```diff
@@ -1,82 +1,87 @@
-.Dd Feb 10, 2018
+.Dd  Aug 25, 2023
```
Comment (wargio) [inline binrz/man/rz-diff.1]: extra useless space

**Review by XVilka — APPROVED** [review]

**Pull request merged**